setup.py

```diff
@@ -28,11 +29,12 @@
     ],
     license='MPLv2',
     name='pyfxa-versions',
-    packages=['pyfxa-versions'],
+    packages=find_packages(),
     package_data={
         "": ["*.txt"],
-        "py-fxa-versions": ["data/py-fxa-versions.json"],
+        "pyfxa-versions": ["data/py-fxa-versions.json"],
     },
+    package_dir={"pyfxa-versions": "pyfxa_versions"},
```

Comment:
This maps the package name to the directory with the package contents in it which helps `find_packages` find all the bits.

https://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=package_dir#using-find-packages

Having said that, now that I'm reading this, I'm not sure what it does. I'll tinker with it and see if we can skip it.

Reply:
wfm